src/PactNet/RequestBuilder.cs

```diff
@@ -241,8 +241,16 @@ IRequestBuilderV3 IRequestBuilderV3.WithJsonBody(dynamic body, JsonSerializerSet
         /// <param name="contentType">Content type override</param>
         /// <returns>Fluent builder</returns>
         IRequestBuilderV3 IRequestBuilderV3.WithJsonBody(dynamic body, JsonSerializerSettings settings, string contentType)
-            => this.WithJsonBody(body, settings, contentType);
-
+            => this.WithJsonBody(body, settings, contentType);

```

Comment:
I think there are generally weird line-ending problems throughout the PR because most changed files say that blank lines have been deleted and added back again. We should check to make sure the line endings are normalised.